OlinkAnalyze/tests/testthat/test-olink_normalization_bridgeable.R

```diff
@@ -0,0 +1,85 @@
+test_that(
+  "olink_normalization_is_bridgeable - works",
+  {
+    expect_message(
+      object = data_explore_check <- olink_norm_input_check(
+        df1 = data_3k,
+        df2 = data_ht,
+        overlapping_samples_df1 = intersect(
+          x = unique(data_3k$SampleID),
+          y = unique(data_ht$SampleID)
+        ) |>
+          (\(x) x[!grepl("CONTROL", x)])() |>
```

Comment:
This is a good point. I did this because this character vector is sent to "olink_norm_input_check" and I wanted to make sure it did not cause any issues in the other function. I would not expect any issues, messages, errors or warnings, but just as an additional step for testing purposes.

Note that in testing it is more acceptable to have a few redundant steps to ensure than things do not go sideways. This is strongly counter-advised in the actual code because it might introduce unexpected behavior of the code.